lib/registrant.rb

```diff
@@ -0,0 +1,26 @@
+class Registrant 
+    attr_reader :name, :age, :permit, :license_data
+
+    def initialize(name, age, permit, license_data)
+        @name = name
+        @age = age
+        @permit = permit
+        @license_data = license_data
+    end
+
+    # def name
+    #    @name = name
+    # end
+
+    # def age
+    #    @age = age
+    # end
+
+    # def permit
+    #    @permit = permit
+    # end
+
+    # def license_data
+    #    @license_data = license_data
+    # end
```

Comment:
Good job recognizing that there are methods you've created with your attr_readers. We did want a method of `def permit?` to be the attr_reader for the `@permit` attribute since it returns a boolean.